### PR TITLE
CSPL-1651: Update documentation

### PR DESCRIPTION
**Pull request by smohan-splunk**
Update documentation on develop to accommodate the following:
Commit https://github.com/splunk/splunk-operator/commit/319978158291637824767bd5699e071d5f964d6a
Commit https://github.com/splunk/splunk-operator/commit/6fabaf08ef36235ba956a5206bc6fef356fa07ba
Commit https://github.com/splunk/splunk-operator/commit/39ff709e0fd69d1b45e962949323af81696614b8
Commit https://github.com/splunk/splunk-operator/commit/a32c15bfaab8d97484875e1ebfa32eb783932436
Commit https://github.com/splunk/splunk-operator/commit/3d4a9d86d3edb1bdd874ebab53890cce5ba4c3f0
PR https://github.com/splunk/splunk-operator/pull/676

### CODE REVIEW

**Review by akondur — APPROVED** [review]

File: config/examples/advanced/c1.yaml [hunk 1]
```diff
@@ -20,27 +20,32 @@ metadata:
...
+    appSources:
+      - name: idxApps
+        location: <apps-location>
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: <apps-path>
+        endpoint: https://<apps-endpoint>
+        secretRef: s3-secret
```
Comment (gaurav-splunk) [inline config/examples/advanced/c1.yaml:37]: Maybe add `region` here as well?

File: config/examples/advanced/c1.yaml [hunk 2]
```diff
@@ -66,3 +71,18 @@ spec:
...
+    appSources:
+      - name: shApps
+        location: <apps-location>
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: <apps-path>
+        endpoint: https://<apps-endpoint>
+        secretRef: s3-secret
```
Comment (gaurav-splunk) [inline config/examples/advanced/c1.yaml:88]: ditto

File: config/examples/advanced/c3.yaml [hunk 3]
```diff
@@ -20,29 +20,34 @@ metadata:
...
+    appSources:
+      - name: idxApps
+        location: <apps-location>
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: <apps-path>
+        endpoint: https://<apps-endpoint>
+        secretRef: s3-secret
```
Comment (gaurav-splunk) [inline config/examples/advanced/c3.yaml:37]: ditto

**Review by gaurav-splunk — APPROVED** [review]

File: config/examples/advanced/c1.yaml (hunk 1 above)
Comment (gaurav-splunk) [reply config/examples/advanced/c1.yaml:37]: Ignore this since this is for develop

**Pull request merged**